Desktop Version

sunga12 opened the pull request:
Desktop version completed

sunga12 committed:
Merge branch 'master' into desktop-home-page

RaoAkif commented on index.html:
```diff
@@ -32,32 +59,43 @@ <h3 class="SD">I'm a Software Developer</h3>
+      <section class="recent-works-heading">
         <h1 class="recent-works-title">My Recent Works</h1>
-      
+        <div class="line"></div>
```
I have observed a few design issues, that could be eliminated using `dislay: none` when you transition from the mobile to the desktop version. To look at, how you can do that, you can follow [this guide.](https://www.w3schools.com/css/css_display_visibility.asp) 

1. So this line:
![image](https://user-images.githubusercontent.com/61361037/213734331-f132bf8d-d887-47b9-8793-4d8714335fbd.png)

is appearing in the desktop version here:
![image](https://user-images.githubusercontent.com/61361037/213734507-1b88a2a4-c9d0-49ff-8848-6923328283a4.png)

2. This line
![image](https://user-images.githubusercontent.com/61361037/213734770-2617b68a-7e58-4f42-a79b-92445f08bba5.png)

in the desktop version appears like this
![image](https://user-images.githubusercontent.com/61361037/213735006-f756e817-07bd-4f94-a40c-f02746d498be.png)

But keep in mind, here in the `form` section, you do not have to use the `display: none` property, you have to switch from the small bold line to this
![image](https://user-images.githubusercontent.com/61361037/213744257-17d74a12-80db-40a9-bc0b-69120907f403.png)

wayungi replied:
- [  ] You have done very good work ✔️ work implementing this request. You however forgot one small detail about the change. Please assign the correct color to that dividing line. Check the Figma design. 👓  The color is 
 ```
background: #DFE1E6;
```

_**Figma**_
![border](https://user-images.githubusercontent.com/99656687/213861548-2c01be5a-dcca-4df2-8151-9a975717189d.png)


 **Your design**
 
![border-2](https://user-images.githubusercontent.com/99656687/213861580-c4f98bba-abc7-4f3c-bef0-249680c44898.png)

RaoAkif commented on index.html:
```diff
@@ -9,7 +9,7 @@
-<header class="home-header">
+<header class="mob-home-header">
     <div class="name logo">
         <div class="name-text" ><a href="" id="logo">Sungabanja Thawethe</a></div>
     </div>
```
[OPTIONAL] You can use your short name here. For example, my full name is `Rao Muhammad Akif Tufail`. But it does look quite expanded. So I use a short one like `Rao Akif`. This is an optional suggestion. If you feel OK with your short name, that's great. By the way, I have also seen some guys, making a text logo, that takes a square or a circular shape, which also looks fine. Think about what feels good for design.

RaoAkif commented on index.html:
```diff
@@ -32,32 +59,43 @@ <h3 class="SD">I'm a Software Developer</h3>
           <a href="https://twitter.com/OfficialAseT"><img src="/images/twitter.png" class="contact-icon"></a>
           <a href="#"><img src="/images/medium.png" class="contact-icon"></a>
         </section>
-    </div>
+  </div>
 </section>
 
     <section class="Recent-works">
-
+      <section class="recent-works-heading">
         <h1 class="recent-works-title">My Recent Works</h1>
-      
+        <div class="line"></div>
```
When I looked at the desktop version, `My Recent Work` section
![image](https://user-images.githubusercontent.com/61361037/213739051-ccfc01e1-58ab-4a95-9f05-40b584c3f0ca.png)
It is behind your `intro` section. Kindly try to meet the FIGMA design requirement.
The purpose of this project is to learn how to successfully switch your design from mobile to desktop design using media queries.

sunga12 replied:
Hi @RaoAkif 

Kindly clarify this comment as I dont quite understand what you meant.

RaoAkif commented on index.html:
```diff
@@ -135,13 +173,17 @@ <h2 class="professional">Professional Art Printing Data</h2>
+    <p class="about-me">Hey there! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
     <button class="resume-link"><a href="#">Get My Resume</a></button>
-  
+    </section>
+
+      <div class="divider"></div>
+
     <section class="lists">
 
```
This is from FIGMA design:
![image](https://user-images.githubusercontent.com/61361037/213742178-2e182dd3-cae5-44d3-a35c-96422497cfca.png)


This is your current bullet points:
![image](https://user-images.githubusercontent.com/61361037/213741978-dc0aa79f-f0c1-4766-8ca2-8bc007b7fafa.png)

You just have tp export these icons, and apply them there.

RaoAkif commented on index.html:
```diff
@@ -203,7 +246,8 @@ <h2 class="contact-comment">I'm always interested in hearing about new projects,
         <li>
           <label for="msg"></label>
-          <textarea id="msg" name="user_message" required maxlength = "500" placeholder="Enter text here"></textarea>  
+
+          <textarea id="msg" name="user_message" required maxlength = "500" placeholder="Hello Sunga. I would like to get in touch with you..."></textarea>  
```
The underlines in the form inputs are missing. You can add them using `border-bottom`

This is your design:
![image](https://user-images.githubusercontent.com/61361037/213743409-04d5b173-d24d-427c-ba67-985ce6ab422a.png)
This is FIGMA design:
![image](https://user-images.githubusercontent.com/61361037/213743285-4d12d3ee-1771-477a-9cb7-1404ad9c68ce.png)

sunga12 replied:
hi @RaoAkif 

I did try to use border-bottom , but each time i did, it brought, the entire border around the text, so I wasnt sure what else to do.

RaoAkif reviewed (changes requested):
## STATUS: REQUIRED CHANGES ♻️

Hi @sunga12,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Required Changes ♻️
#### Please remove the branch conflicts before taking another review, otherwise it would be marked as InValid
![image](https://user-images.githubusercontent.com/61361037/213749672-45bf21fe-9618-40cb-8f10-ae5f71c7d9be.png)

_Check the comments under the review._
<img src="https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966" />

## Optional suggestions
- Remove all the excess files from the project that you might be working on to match the design. If you want to work with them for testing or other reason, you can create another branch, or altogether another similar project locally, whatever you feel comfortable with.
- _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sunga12 committed:
Merge branch 'master' into desktop-home-page

sunga12 committed:
Update according to Review

sunga12 committed:
Fix linter errors

sunga12 committed:
Delete all unnecessary files

sunga12 commented:
Required changes to be made:

- Resolve all branch conflicts
- Remove all unnecessary files
- Reposition the Recent works page to eliminate overlap
- Reposition the Horizontal bar and display it correctly in both mobile and desktop version
- Use shorthand name for logo
- Change bullet lists on the about me page to match the Figma template
- Add a bottom border to form inputs

Changes made:

- Resolved all branch conflicts
- Removed all unnecessary files
- Repositioned the Recent works page to eliminate overlap
- Repositioned the Horizontal bar on the about me page and display it correctly in both mobile and desktop versions
- Used shorthand name for logo
- Changed bullet lists on the about me page to match the Figma template
- Added a bottom border to form inputs

wayungi commented on index.html:
```diff
@@ -62,7 +62,7 @@ <h3 class="SD">I'm a Software Developer</h3>
 </section>
 
-    <section class="Recent-works">
+    <section class="recent-works">
```
- [ ] Please Fix the icons. You need to add a `.` to the beginning of the icon file path as shown below `<a href="#"><img src="./images/medium.png" class="contact-icon"></a>` You should always start your relative path with a `.` You can read more about it [here](https://stackoverflow.com/questions/66887694/in-the-file-path-what-does-mean) 
![links-1](https://user-images.githubusercontent.com/99656687/213862339-be942d3f-4a82-4275-a105-49f18e3aedf5.png)

wayungi reviewed (changes requested):
Hi <STUDENT_MENTION>, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![image](https://user-images.githubusercontent.com/99656687/213862399-56410977-212e-4c60-bd94-e4308c822a8e.png)
- [ ] Good PR title
- [ ] GitHub flow
- [ ] Descriptive commit message
- [ ] Professional README
- [ ] All linter checks passing


 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sunga12 committed:
Update according to 2nd review

sunga12 commented:
Changes Required

- Change the path for social media icons
- Change colour for the horizontal line in the contact page to DFE1E6

Changes made:

- Changed the path for social media icons
- Changed colour for the horizontal line in the contact page to DFE1E6

AlphaNtihinduka commented on index.html:
```diff
@@ -62,7 +62,7 @@ <h3 class="SD">I'm a Software Developer</h3>
 </section>
 
-    <section class="Recent-works">
+    <section class="recent-works">
       <section class="recent-works-heading">
         <h1 class="recent-works-title">My Recent Works</h1>
         <div class="line"></div>
```
- [ ] [OPTIONAL] Please ensure that you follow the Figma design template. I have noticed that the `padding-top` is large in the recent section. Kindly ensure to reduce that to make the project look professional ☺️ 

Your design 😎 
![up rece1](https://user-images.githubusercontent.com/99394374/213865625-80753503-36af-4e46-84ad-8e56a14986f0.PNG)

Figma design 😎 
![up rece2](https://user-images.githubusercontent.com/99394374/213865636-81c929a6-7442-4409-a906-fce5809d2efd.PNG)

AlphaNtihinduka commented on styles.css:
```diff
@@ -1246,7 +1248,7 @@
   }
 
   .proj-desc {
```
- [  ] [OPTIONAL] I have noticed that the `project description` is overlapping the subtitle 😞 . Kindly ensure that the description is independent.

Your design 😎 
![overlapping](https://user-images.githubusercontent.com/99394374/213866110-2364239c-fa0b-43d9-837e-758175380520.PNG)

AlphaNtihinduka reviewed (approved):
# STATUS: APPROVED :green_circle: :clap: 
## Hi @sunga12 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

![approved-accepted](https://user-images.githubusercontent.com/99394374/212473432-e725752b-8682-4dbb-993e-9e466dbe7acb.gif)

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

### Danger Zone ❌ 
- [ ] [OPTIONAL] However much this is marked optional, kindly ensure to implement the suggestions before proceeding 🚶‍♂️ . I have noticed that there is a horizontal scroll on your webpage. The experience of scrolling horizontally can make the screen judder and create a headache-inducing feeling 😞 . 
![resp](https://user-images.githubusercontent.com/99394374/213865845-a6787b1d-cb5a-4fc5-b488-c49099e1d58e.PNG)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:
